tests/ensure_paths_are_short.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python2
```

Comment:
Is this because of RHEL6 builds? I think that this is a script that can run both Python 2 and Python 3, but do we want to be backwards-compatible?

Reply:
The script can run with both python2 and python3. What do you suggest the shebang should be?